website/management/commands/recalc_all_leaderboards.py

```
@@ -0,0 +1,81 @@
+            try:
+                with transaction.atomic():
+                    # Update streak first
+                    profile.update_streak_and_award_points()
+
+                    # Calculate score
+                    score, breakdown = LeaderboardScoringService.calculate_for_user(user)
+
+                    profile.leaderboard_score = score
+                    profile.quality_score = breakdown.get("goals", 0)
+                    profile.check_in_count = DailyStatusReport.objects.filter(user=user).count()
+                    profile.last_score_update = timezone.now()
```

Ruff (0.14.7)

Undefined name `timezone`

(F821)